dmx/README.md

```diff
@@ -240,3 +254,32 @@ result = {
 print(result)
 ```
 
+### Note:
+
+Below is UDS task create in JSON format. "dmxOption" is an optional parameter that gets
+passed into DMX script if provided.
+
+```
+{
+  "category": "INGEST",
+  "filter": "string",
+  "sourceUri": "string",
+  "destinationUri": "string",
+  "sourceCredentials": {
+    "accessKey": "string",
+    "secretKey": "string"
+  },
+  "destinationCredentials": {
```

Comment:
We should take out references to sourceCredentials and destinationCredentials in any API docs, since it's just supported for backwards compatability and shouldn't be used by anyone (unless I'm mistaken), maybe we should make a ticket for taking out all those references to it in our other API docs?